### DISCUSSION TITLE
I don't add package npm

### DISCUSSION THREAD

**Discussion by naphattharawat** [npm/feedback#434] [Website] [answered]
I don't add package.it alert package notfound or permission   but me is owner organzation
![image](https://user-images.githubusercontent.com/29243602/121621106-b75a0500-ca95-11eb-96ef-b04ccd357d2f.png)

**Comment by MylesBorins** [comment, marked as answer]
@naphattharawat  it looks like the @mophtan/test package doesn't exit
